spec/features/staff/users_can_create_a_planned_disbursement_spec.rb

```diff
@@ -108,6 +111,8 @@
       )
 
       expect(page).to have_content t("activerecord.errors.models.planned_disbursement.attributes.financial_quarter.in_the_past")
+
+      travel_back
```

Comment:
I don't have much experience using Rails' `travel_to` functionality, so I have a question about this: If we don't use `travel_back`, would we still be in the same time frame for the rest of the tests?

Reply:
In some cases, yeah, so it's always good to get in the habit of travelling back after you've stubbed the time